packages/wallet/src/redux/protocols/funding/player-a/__tests__/scenarios.ts

```diff
@@ -22,12 +23,17 @@ import { EMPTY_SHARED_DATA } from '../../../../state';
 // ---------
 const processId = 'process-id.123';
 const sharedData = EMPTY_SHARED_DATA;
+const strategy = Strategy.IndirectFunding;
+const targetChannelId = '0x123';
+const opponentAddress = '0xf00';
 
 const props = {
   processId,
   sharedData,
   fundingState: 'funding state' as 'funding state',
-  targetChannelId: '0x123',
+  targetChannelId,
+  opponentAddress,
```

Comment:
I think this makes sense to do at some point but I'm wondering if we should hold off and do this consistently through our protocol reducers?